questions.md

```diff
@@ -0,0 +1,4 @@
+### Pull Requests
+
+* What's the difference between a "pull request" and a "branch"?
```

Comment:
Hello @amightyo! 
A pull request is a collaboration feature where you are asking for review from members of your team on one branch (or several!). Pull requests are a GitHub feature.

A branch is a pointer to a commit, and a Git feature.

Reply:
Thank you! 👍